### PR TITLE
BLADE-474 Properly detect workspace version from gradle model

### PR DESCRIPTION
**Pull request by christopherbryanboyd**

### CODE REVIEW

File: cli/src/main/java/com/liferay/blade/cli/gradle/GradleWorkspaceProvider.java [hunk 1]
```diff
@@ -97,18 +98,55 @@ public boolean isDependencyManagementEnabled(File dir) {
 
 			Matcher matcher = patternWorkspacePluginVersion.matcher(settingsGradleFileContent);
 
-			if (!targetPlatformEnabled || !matcher.find()) {
+			boolean foundMatch = matcher.find();
```
Comment (lawrence-lee) [inline cli/src/main/java/com/liferay/blade/cli/gradle/GradleWorkspaceProvider.java]: Is it okay to get rid of this?

**Comment by gamerson** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by gamerson** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by gamerson** [comment]
Pull request submitted to https://github.com/gamerson/liferay-blade-cli/pull/901. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**